### PR TITLE
IMB-88: Change database configuration for IMA

### PR DESCRIPTION
**Pull request by mislam987**
## What
- Change database configuration for IMA (replacing UAN with CEPR, and adding the 'duty to remove' column) [IMB-88](https://collaboration.homeoffice.gov.uk/jira/browse/IMB-88)

## Why
- To use CEPR instead of UAN
- To enable the ban-only/DTR forking in the IMA form

## How
- Change 'uan' to 'CEPR' in /services/ima/db_tables_config.json
- Add migration files for replacing 'uan' with 'CEPR' in /services/ima/migrations

## Testing
- Tests passing locally

### CODE REVIEW

File: services/ima/migrations/20240202143333_rename_uan_column_to_cepr.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+exports.up = function(knex) {
+  return knex.schema.table('saved_applications', table => {
+    table.renameColumn('uan', 'CEPR');
```
Comment (jamiecarterHO) [inline services/ima/migrations/20240202143333_rename_uan_column_to_cepr.js:4]: If sticking with lowercase, 'CEPR' should be lowercase too.

**Review by jamiecarterHO — COMMENTED** [review]
Couple of questions around uppercasing 'CEPR' and what the rollback function would do to the column rename.

File: services/ima/migrations/20240202143333_rename_uan_column_to_cepr.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  });
+};
+
+exports.down = function(knex) {
+  return knex.schema.table('saved_applications', table => {
+    table.dropColumn('CEPR');
```
[the remark this answers is not in the document]
Comment (mislam987) [reply services/ima/migrations/20240202143333_rename_uan_column_to_cepr.js]: That makes sense. I've changed it to rename the column to 'uan'.

File: services/ima/db_tables_config.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   {
     "tableName": "saved_applications",
     "modelName": "postgres-model",
-    "additionalGetResources": ["uan", "date_of_birth", "email", "submitted_at"],
+    "additionalGetResources": ["CEPR", "date_of_birth", "email", "submitted_at", "duty_to_remove_alert"],
```
[the remark this answers is not in the document]
Comment (mislam987) [reply services/ima/db_tables_config.json]: Changed it to lower case for consistency.

File: services/ima/migrations/20240202143333_rename_uan_column_to_cepr.js (hunk 1 above)
Comment (mislam987) [reply services/ima/migrations/20240202143333_rename_uan_column_to_cepr.js]: Changed to lowercase too.

**Pull request merged**